_tutorials/page10.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: tutorial
+title: Include
+name: Include
+previous_page_title: Default Values
+next_page_title: Project Name
+---
+
+# Include
+
+Using the `!include` notation, you can [split your specification into multiples files][docs-include].
```

Comment:
Can we recursively include files? Can the snippet file be composed of several other includes? Useful to add a note hinting that.